### ISSUE TITLE
Fail to connect to database

### ISSUE THREAD

**Issue by Zepeng-Mu** [boxiangliu/locuscomparer#11]
Hi,

While running `locuscompare` function I encountered this error:
```
Error in .local(drv, ...) :
  Failed to connect to database: Error: Unknown MySQL server host 'locuscompare-us-west-2a.cvocub39nnri.us-west-2.rds.amazonaws.com' (2)
Calls: locuscompare ... get_position -> <Anonymous> -> <Anonymous> -> .local
Execution halted
Warning message:
In rm(config) : object 'config' not found
```
I did not have this problem when I was doing the same thing before, though.

Thanks

**Comment by boxiangliu** [comment]
Hello Zepeng, 

Sorry for the delay. This is an Amazon AWS database connection error. I just tried on my laptop but I am not able to reproduce this error from my end. 

Can you try again? The AWS database will occasionally become disconnected but later fixed itself.

**Comment by Zepeng-Mu** [comment]
Hi,

I actually noticed that this is due to an error in my code, which resulted in an empty input GWAS file. But I have no idea why the error looked like an AWS error.

After fixing the error in my code I did not get this error anymore.

I once ran the function on a cluster without the Internet, so that it cannot connect to the database, but I remember it was another error message (I might wrong though).

But I have already solved the problem, although I do not know what this error message implicates.
Thanks!
